apps/client/electron/tsconfig.json

```diff
@@ -0,0 +1,6 @@
+{
+  "extends": "../../../tsconfig.json",
+  "compilerOptions": {
+    "outDir": "../../../build/electron"
```

Comment:
This should eventually change to something client-scoped. The manager is also Electron.